app/views/custom_fields/_custom_options.html.erb

```diff
@@ -111,7 +143,10 @@ See COPYRIGHT and LICENSE files for more details.
               <%= link_to '',
                           delete_option_of_custom_field_path(id: custom_field.id || 0, option_id: custom_option.id || 0),
                           method: :delete,
-                          data: { confirm: t(:'custom_fields.confirm_destroy_option') },
+                          data: {
+                            action: 'admin--custom-fields#removeOption',
+                            confirm: t(:'custom_fields.confirm_destroy_option')
```

Comment:
This UJS call is not changed, it has a replacement using turbo (`turbo_confirm`) but that's not part of our stack yet. So this should be replaced separately.